simplerisk/includes/config.php

```diff
@@ -21,4 +21,5 @@
 // Use database for sessions
 define('USE_DATABASE_FOR_SESSIONS', 'true');
 
-?>
+// Path to the cert to be used for SSL connections to the database
+define('DB_SSL_CERTIFICATE_PATH', '');
```

Comment:
Is the plan to pull this from an env var?

Reply:
Yes, otherwise default it to an empty string